src/DynamoCore/DynamoCore.csproj

```diff
@@ -286,6 +286,7 @@ limitations under the License.
     <Compile Include="Search\SearchDictionary.cs" />
     <Compile Include="Utilities\ResourceLoader.cs" />
     <Compile Include="Visualization\RenderPackageCache.cs" />
+    <EmbeddedResource Include="BuiltInAndOperators\BuiltInImages.resx" />
```

Comment:
I'm confused by this... why do we need to resxs for images?

Reply:
Dictionary icons .resx should be in `Builtin.csproj` not `DynamoCore.csproj` I think since all the Dictionary work lives in `Builtin.csproj`